src/vcpkg/tools.cpp

```diff
@@ -579,6 +579,24 @@ gsutil version: 4.58
             });
         }
 
+        const PathAndVersion get_system_tool_path(const VcpkgPaths& paths, const std::string& tool) const
+        {
+            auto& fs = paths.get_filesystem();
+            if (get_environment_variable("VCPKG_FORCE_SYSTEM_BINARIES").has_value())
+            {
+                auto tool_path = fs.find_from_PATH(tool);
+                if (tool_path.empty())
+                {
+                    Checks::exit_with_message(VCPKG_LINE_INFO, "Error: Couldn't find %s on the path", tool);
+                }
+                return {tool_path[0], "0"};
+            }
+            else
+            {
+                return get_path(paths, CMakeProvider());
```

Comment:
This is probably wrong; you need `CMakeProvider` to be `ToolProvider` where `Tool` is one of `Git`, or `Ninja`, or w/e